test/container/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM redis:5.0.8 as redis
+FROM redis:5.0.10 as redis
```

Comment:
Can't use alpine image here, since redis binaries are not linked statically (and we copy them later on for local execution within the Docker image)